etcdctl/ctlv3/command/util.go

```diff
@@ -31,7 +31,9 @@ func printKV(isHex bool, kv *pb.KeyValue) {
 		v = addHexPrefix(hex.EncodeToString(kv.Value))
 	}
 	fmt.Println(k)
-	fmt.Println(v)
+	if v != "" {
```

Comment:
we should not do this since some key might have empty value. I feel just printing empty as value when user explicitly asks for only keys is fine.